src/init/AppRunner.ts

```diff
@@ -85,11 +85,11 @@ export class AppRunner {
 
     // Gather settings for instantiating the server
     const loaderProperties: IComponentsManagerBuilderOptions<App> = {
-      mainModulePath: this.resolveFilePath(params.mainModulePath),
+      mainModulePath: resolveAssetPath(params.mainModulePath),
       dumpErrorState: true,
       logLevel: params.loggingLevel as LogLevel,
     };
-    const configFile = this.resolveFilePath(params.config, 'config/default.json');
+    const configFile = resolveAssetPath(params.config ?? `${modulePathPlaceholder}/config/default.json`);
```

Comment:
We should just have this as a default option in the yargs settings above. I don't think there's any reason not to have it there (and that way it will show up as the default value when calling with `--help`).

Reply:
Yeah; this file was created before the existence of `resolveAssetPath`, and I notice and can now be simplified a lot.